CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 # Contributing
 
-If you want to add to arboretum, please familiarise yourself with the code & our [Coding Standards][]. Before you submit a PR, please [file an issue][new collab] to request collaborator access.
+If you want to add to arboretum, please familiarize yourself with the code & our [Coding Standards][]. Make a fork of the repository & file a Pull Request from your fork with the changes. You will need to click the checkbox in the template to show you agree to the [Developer Certificate of Origin](../blob/master/DCO1.1.txt).
```

Comment:
Should the link be `../blob/main/DCO1.1.txt`?

Reply:
yes - thank you!

Reply:
fixed in https://github.com/ComplianceAsCode/auditree-arboretum/pull/30/commits/80c72925f15e2823f7e0a6f494011d0efc12f187